### DISCUSSION TITLE
The battery level reported via Bluetooth, is it only for the left half?

### DISCUSSION THREAD

**Discussion by ecerulm** [KinesisCorporation/Adv360-Pro-ZMK#139] [Q&A] [answered]
on macOS > System Preferences > Bluetooth I can see the battery level % for Adv360 Pro 

<img width="484" alt="image" src="https://user-images.githubusercontent.com/58676/231404563-ea735918-ad55-4a0b-907a-7716a4f0fff9.png">

but it's that the battery level for the left half of the keyboard only? or it's both combined? I does look like it's only for the left side to me. 

Anybody knows how it's actually calculated and if it's possible to have the battery level for each half separately like they do for the Airpod Pro?

**Comment by ecerulm** [comment]
From https://zmk.dev/docs/features/battery#:~:text=If%20your%20keyboard%20has%20a,side%20is%20reported%20over%20bluetooth.

> For split keyboards, only the battery level of the central (usually left) side is reported over bluetooth.

**Comment by ReFil** [comment, marked as answer]
Only the central battery is reported to the OS, and battery reporting is known to be inconsistent and is set to be disabled in a future release, use the mod+hk2 or mod+hk4 shortcut to make the leds report the battery state, each half will report it's own battery state
